### PR TITLE
Create Rule “managing-errors-with-code-auditor/rule”

### PR DESCRIPTION
**Pull request by BrookJeynes**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/managing-errors-with-code-auditor/rule.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+type: rule
+title: Do you know how to manage errors with Code Auditor
+uri: managing-errors-with-code-auditor
+authors:
+  - title: Brook Jeynes
+    url: https://www.ssw.com.au/people/brook-jeynes/
+created: 2023-06-05T05:51:53.697Z
+guid: df5dca2b-077b-4f09-bb1a-46c3ae0cd2da
+---
+Code auditing is an essential practice that empowers developers, quality assurance teams, and organizations to identify and rectify potential flaws, weaknesses, and security risks within their codebase
```
Comment (william-liebenberg) [inline rules/managing-errors-with-code-auditor/rule.md:11]: need full stop :)

File: rules/managing-errors-with-code-auditor/rule.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+<!--endintro-->
+
+Every sprint, some time should be devoted to resolving Code Auditor errors and warnings. To aid in this, long-lasting PBI items should be created and carried over each sprint keeping a history of the work done.
```
Comment (william-liebenberg) [inline rules/managing-errors-with-code-auditor/rule.md:15]: Add a link to Code Auditor site

File: rules/managing-errors-with-code-auditor/rule.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+title: Do you know how to manage errors with Code Auditor
+uri: managing-errors-with-code-auditor
+authors:
+  - title: Brook Jeynes
+    url: https://www.ssw.com.au/people/brook-jeynes/
+created: 2023-06-05T05:51:53.697Z
+guid: df5dca2b-077b-4f09-bb1a-46c3ae0cd2da
+---
+Code auditing is an essential practice that empowers developers, quality assurance teams, and organizations to identify and rectify potential flaws, weaknesses, and security risks within their codebase
+
```
Comment (william-liebenberg) [inline rules/managing-errors-with-code-auditor/rule.md:12]: You should try to announce Code Auditor in the intro - with a link to the site.

**Review by william-liebenberg — CHANGES_REQUESTED** [review]

File: rules/managing-errors-with-code-auditor/rule.md (hunk 2 above)
Comment (william-liebenberg) [reply rules/managing-errors-with-code-auditor/rule.md:15]: if you add a link in the intro, then don't worry about this one.

**Review by william-liebenberg — APPROVED** [review]

File: rules/managing-errors-with-code-auditor/rule.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Code auditing is an essential practice that empowers developers, quality assurance teams, and organizations to identify and rectify potential flaws, weaknesses, and security risks within their codebase.
+
+[SSW Code Auditor](https://codeauditor.com/) is the perfect tool to audit your code helping you find:
+- Broken links
+- HTML errors
+- Google Lighthouse issues
+
+<!--endintro-->
+
+Every sprint, some time should be devoted to resolving Code Auditor errors and warnings. To aid in this, long-lasting PBI items should be created and carried over each sprint keeping a history of the work done.
```
Comment (wicksipedia) [inline rules/managing-errors-with-code-auditor/rule.md:20]:
```suggestion
Every Sprint, some time should be devoted to resolving Code Auditor errors and warnings. To aid in this, long-lasting PBI items should be created and carried over each sprint keeping a history of the work done.
```

**Review by wicksipedia — APPROVED** [review]

**Pull request merged**